content/pages/sprint/2021/11late-taipei.rst

```diff
@@ -11,7 +11,7 @@ Scisprint 2021 Late November in Taipei
 
 We cordially invite you to join the short sprint for `solvcon
 <https://solvcon.net/>`_ projects.  solvcon_ is a host of projects focusing on
-numerically solving the governing equations of transient phenomena.
+numerically solving the governing equations of transient phenomena.The 2 projects we share are partions of solvcon_ and we would assume you have no domain knowledge of equation solving. Anyone who have some fundamental unix and program skill are welcomed
```

Comment:
Please keep a space after the period.

Reply:
It will be even better if you can keep the number of whitespaces consistent with other places in the file.

Reply:
update the committed content. but don't get the idea of 2nd comment.(sorry).

Reply:
LGTM.

The file use two whitespaces before the period of the previous sentence and you did not follow it.  You did not break lines in a concistent way either.  But these are minor.